src/web/components/SignInGate/gateDesigns/design-opt-test/SignInGateDesignOptVar5.tsx

```diff
@@ -0,0 +1,262 @@
+import React, { FunctionComponent } from 'react';
+import { css, cx } from 'emotion';
+
+import {
+	headline,
+	body,
+	titlepiece,
+} from '@guardian/src-foundations/typography';
+import { from } from '@guardian/src-foundations/mq';
+import { space, palette } from '@guardian/src-foundations';
+import { LinkButton } from '@guardian/src-button';
+import { Link } from '@guardian/src-link';
+import { cmp } from '@guardian/consent-management-platform';
+import { trackLink } from '@frontend/web/components/SignInGate/componentEventTracking';
+import { SerializedStyles } from '@emotion/core';
+import { SignInGateProps } from '../types';
+import {
+	actionButtons,
+	bodyText,
+	firstParagraphOverlay,
+	hideElementsCss,
+	laterButton,
+	privacyLink,
+	registerButton,
+	signInGateContainer,
+} from '../shared';
+
+interface LinesProps {
+	cssOverrides?: SerializedStyles;
+}
+
+export const Lines: FunctionComponent<LinesProps> = (props) => {
+	const { cssOverrides } = props;
+	const n = 3;
+	const thickness = 1;
+	const distance = 6;
+	const height = (n - 1) * distance + thickness;
+	const color = palette.brand[400];
+
+	const linesCss = css`
+		background-image: repeating-linear-gradient(
+			to bottom,
+			${color},
+			${color} ${thickness}px,
+			transparent ${thickness}px,
+			transparent ${distance}px
+		);
+		background-repeat: 'repeat';
+		background-position: 'top';
+		height: ${height}px;
+		border: 0;
+		margin: 0;
+		opacity: 0.2;
+		width: calc(100% + 20px);
+		padding-left: 20px !important;
+		margin-left: -20px !important;
+		${from.leftCol} {
+			width: calc(100% + 10px);
+			padding-left: 10px !important;
+			margin-left: -10px !important;
+		}
+		${cssOverrides}
+	`;
+
+	return <hr className={linesCss} />;
+};
+
+const signInGateBackground = css`
+	/* stylelint-disable-next-line color-no-hex */
+	background-color: #eaf1fd;
+`;
+
+const headingStyles = css`
+	${titlepiece.small({})};
+	border-top: 2px ${palette.brandText.ctaPrimary} solid;
+	padding-bottom: ${space[9]}px;
+	color: ${palette.brandText.ctaPrimary};
+
+	${from.phablet} {
+		padding-right: 160px;
+		${headline.medium({ fontWeight: 'bold' })};
+	}
+`;
+
+const bodyBold = css`
+	${body.medium({ fontWeight: 'bold' })}
+	padding-bottom: 20px;
+	color: ${palette.brandText.ctaPrimary};
+	${from.phablet} {
+		padding-right: 110px;
+	}
+`;
+
+const leftMarginFill = css`
```

Comment:
looks like the styling could be improved for this experiment 5. We could leave it for now since it should be temporary. What do you think @coldlink ?

Reply:
yeah, it's really fiddly to get the design to match up exactly due to padding and margins elsewhere, especially with the change to the background colour, and slight changes become more pronounced

for now, we'll leave it as is for the test, if it becomes the winner, we can do more work properly implementing it